internal/pkg/cli/secret_init.go

```diff
@@ -148,6 +148,13 @@ func (o *secretInitOpts) Validate() error {
 		if err != nil {
 			return fmt.Errorf("get application %s: %w", o.appName, err)
 		}
+		if o.values != nil {
```

Comment:
:woman-facepalming:

Reply:
What do you think we leave this chunk at its original place and modify `o.targetEnv` instead?
```go
func (o *secretInitOpts) targetEnv(envName string) (*config.Environment, error) {
        if o.appName == "" {
                return nil, nil
        }
 	env, err := o.store.GetEnvironment(o.appName, envName)
	if err != nil {
		return nil, fmt.Errorf("get environment %s in application %s: %w", envName, o.appName, err)
	}
	return env, nil
}
```

Reply:
Umm I think it still prefer handling it in the validate, since it would make more sense when reading `validate`